datafusion/physical-plan/src/sorts/sort.rs

```diff
@@ -697,7 +709,7 @@ fn write_sorted(
         writer.num_rows,
         human_readable_size(writer.num_bytes as usize),
     );
-    Ok(())
+    Ok(writer.num_rows)
```

Comment:
Not directly related but I wonder why `IPCWriter` has `num_rows` as `u64`. I feel it is more natural to have it as `usize` because it is accumulating in `self.num_rows += batch.num_rows() as u64;` in `IPCWriter.write()`

Reply:
Thanks for the review. This is good point. I think it is also valid for `num_batches` and `num_bytes` and this requires redundant casting on `IPCWriter.write()` and `ExternalSorter.spill()`. I plan to submit separate PR for this.

Reply:
Created #9937 for the fix.